ansible/files/runtimes.json

```diff
@@ -262,7 +248,7 @@
                 "image": {
                     "prefix": "openwhisk",
                     "name": "action-ballerina-v0.990.2",
-                    "tag": "nightly"
+                    "tag": "nightly" // Todo: Need to use a fixed tag after releasing the runtime
```

Comment:
I noticed there is no released version of this runtime and it's even not listed in the website.
Should we release this runtime first?

Reply:
I think we should probably drop ballerina from 1.0.0 branch, but I will defer to @rabbah.  I think ballerina 1.2.x is out now, so if we did want to support it, the "right" thing to do is update the runtime to a newer version of ballerina, then release it, then use that tag.   I don't think we should hold 1.0.0 of OpenWhisk to do that.